sites/frontend/components/Header/Nav/SubNav.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import { styled } from '@guardian/guui';
+import type { LinkType } from './__config__';
+
+type Props = {
+    parent?: LinkType,
+    links: Array<LinkType>,
+};
+
+const UlStyled = styled('ul')({
+    listStyle: 'none',
+    li: {
+        float: 'left',
+    },
+    a: {
+        display: 'block',
+        fontWeight: 400,
+        color: '#121212',
+        padding: '0 0.3125rem',
+        fontSize: '1rem',
+        height: '2.625rem',
+        lineHeight: '2.625rem',
+        textDecoration: 'none',
+    },
+
+    'a.parent': {
+        fontWeight: 'bold',
+    },
+});
+
+const SubNav = ({ parent, links }: Props) => {
+    let lis = [];
+
+    if (parent) {
+        const parentLink = (
+            <li>
+                <a className="parent" href={parent.url}>
+                    {parent.title}
+                </a>
+            </li>
+        );
+
+        lis.unshift(parentLink);
+    }
+
+    lis = lis.concat(
+        links.map(link => (
+            <li>
```

Comment:
`React` will likely show a warning in the console that 'Each child in an array or iterator should have a unique "key" prop'